addons/stock/models/stock_move.py

```diff
@@ -714,6 +715,9 @@ def _assign_picking(self):
                 move.recompute()
         return True
 
+    def _picking_post_process(self, picking):
```

Comment:
just a matter of clarity, maybe a less generic name, e.g. `_assign_picking_post_process`